frontend/cypress/integration/talentsearch/direct-intake.spec.js

```diff
@@ -56,6 +56,7 @@ describe("Talentsearch Direct Intake Page", () => {
       cy.visit('/en/browse/pools');
 
       cy.location('pathname').should('eq', '/en/talent/profile/create-account');
+      cy.contains("successfully logged in");
```

Comment:
Good catch. If it doesn't feel out-of-scope, consider using this as opp to add an aria role (whichever one fits best) to the "notification" in which your text is found, and using that instead of a whole page match.

```suggestion
      cy.findByRole('status', /successfully logged in/i);
```